workitem.go

```diff
@@ -98,7 +101,11 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 // Create runs the create action.
 func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
-		wi, err := appl.WorkItems().Create(ctx.Context, ctx.Payload.Type, ctx.Payload.Fields)
+		currentUser, err := login.ContextIdentity(c.tokenManager, ctx)
```

Comment:
Yes @aslakknutsen  , that sounds a better way instead of passing around the tokenManager.

Different question I had - When your pushed some fixes yesterday, how did you do that ? You pushed to my fork's branch, right ? Did you squash, I found it very difficult to exactly see your changes only. Where can I find that one commit.